src/mn_setup.c

```diff
@@ -3412,6 +3432,19 @@ void MN_General(int choice)
     current_menu = gen_settings[current_page];
     current_tabs = gen_tabs;
     SetupMenu();
+
+    if (freelook && direct_vertical_aiming)
```

Comment:
Maybe force `menu_freelook` to off if blocked.

Reply:
This is done by setting a limit on the `freelook_mode` variable.